tests/README.md

```diff
@@ -0,0 +1,202 @@
+# Cloudbeat Automated Tests Framework (C-ATF)
+> 
+
+This project provides an ability to develop component and integration tests for Cloudbeat.
+
+
+## Getting started
+
+All installations below (except of docker desktop) are available via [Homebrew](https://brew.sh/).
+If you have a different OS, in order to install, please go forward through instructions provided in links. 
+
+1. Install [Docker Desktop](https://docs.docker.com/desktop/mac/install/)
+
+
+2. Install [JUST](https://github.com/casey/just) command runner
+
+```shell
+brew install just
```

Comment:
pay attention that `brew` is available only on mac machines
this guideline should serve all user's platforms.

Reply:
As it is written in Getting started section, users that are no working on mac machines shall go forward through provided links and find relevant installation command / package for their os platform.